code/obj/item/kendo.dm

```diff
@@ -0,0 +1,252 @@
+
+/obj/item/clothing/gloves/kote
+    name = "kote (小手)"
+    desc = "Big poofy gloves to cover the hands in kendo sparring."
+    icon_state = "kote"
+    item_state = "kote"
+    material_prints = "navy blue, synthetic leather fibers"
+    crit_override = 1
+    bonus_crit_chance = 0
+    stamina_dmg_mult = 0.35
+
+    setupProperties()
+        ..()
+        setProperty("coldprot", 7)
+        setProperty("conductivity", 0.3)
+
+//======
+//Shinai
+//======
+
+/obj/item/shinai
+    name = "shinai (竹刀)"
+    desc = "A sword-like weapon made of slats of bamboo. Shinai are made to reflect the weight of a katana, but disperse impact on hit to minimize damage."
+    icon = 'icons/obj/items/weapons.dmi'
+    inhand_image_icon = 'icons/mob/inhand/hand_weapons.dmi'
+    icon_state = "shinai"
+    item_state = "shinai-light"
+
+    w_class = 4.0
+    two_handed = 1
+    throwforce = 4.0
+    throw_range = 4
+    stamina_crit_chance = 2
+
+    //these combat variables will change depending on the guard
+    force = 6.0
+    stamina_damage = 10
+    stamina_cost = 5.0
+
+    hit_type = DAMAGE_BLUNT
+    flags = FPRINT | TABLEPASS | USEDELAY
+    c_flags = EQUIPPED_WHILE_HELD
+    item_function_flags = USE_INTENT_SWITCH_TRIGGER | USE_SPECIALS_ON_ALL_INTENTS
+
+    var/guard
+
+    proc/change_guard(var/mob/user,var/intent)
+        user.do_disorient(10,0,0,0,0,0,null)
+        guard = intent
+        switch(guard)
+            if("help")
+                force = 5
+                stamina_damage = 10
+                stamina_cost = 5
+                item_state = "shinai-light"
+                src.setItemSpecial(/datum/item_special/simple/kendo_light)
+            if("disarm")
+                force = 6
+                stamina_damage = 10
+                stamina_cost = 8
+                item_state = "shinai-sweep"
+                src.setItemSpecial(/datum/item_special/swipe/kendo_sweep)
+            if("grab")
+                force = 7
+                stamina_damage = 15
+                stamina_cost = 10
+                item_state = "shinai-thrust"
+                src.setItemSpecial(/datum/item_special/rangestab/kendo_thrust)
+            if("harm")
+                force = 8
+                stamina_damage = 30
+                stamina_cost = 35
+                item_state = "shinai-heavy"
+                item_state = "shinai-heavy"
+                src.setItemSpecial(/datum/item_special/simple/kendo_heavy)
+        user.update_inhands()
+        src.buildTooltipContent()
+
+    proc/parry_block_check(var/mob/living/carbon/human/attacker,var/mob/living/carbon/human/defender)
+        if(attacker == defender)
+            return
+
+        if((attacker.a_intent == defender.a_intent) && !defender.hasStatus("disorient"))
+            playsound(defender, "sound/impact_sounds/kendo_parry_[pick(1,2,3)].ogg", 50, 1)
+            attacker.do_disorient()
+            SPAWN_DBG(1 SECOND)
+                attacker.delStatus("disorient")
```

Comment:
This method of applying/removing disorients works but is improper and gives the player the wrong countdown at the top-right of their screen since it shows a 6 second disorient that deletes after 1 second. 

You should just call the do_disorient proc with the disorient argument set to what you want it (10 deciseconds), instead of using a spawn and then deleting the disorient status effect.

Reply:
Fixing! :heart: